### ISSUE TITLE
English version

### ISSUE THREAD

**Issue by theel0ja** [ElysiumSolutions/Tapaaminen.net#39]
Feature request - Localization support

**Comment by MarkoKaartinen** [comment]
Yeah, this is a possibility. I had plans to do a separate english site with domain etc.
I only need to come up with a good domain name and do some coding/translating and it should be quite fine. 

This is in my radar.
